Formula/rpm.rb

```diff
@@ -34,6 +34,13 @@ class Rpm < Formula
   depends_on "gettext"
   depends_on "xz"
   depends_on "ossp-uuid"
+  # use external libraries, until the internal have been fixed
+  depends_on "lua"
+  depends_on "syck"
+  depends_on "openssl"
```

Comment:
The underlying issue with internal lua has been checked in, syck will be fixed as soon as I see the reproducer. Meanwhile rpm5 was designed to use either internal/external versions of syck/lua (and neon/BeeCrypt/popt/pcre) depending on build configuration.